media-gfx/blender/blender-2.83.15.ebuild

```diff
@@ -57,9 +59,9 @@ RDEPEND="${PYTHON_DEPS}
 	virtual/opengl
 	alembic? ( >=media-gfx/alembic-1.7.12[boost(+),hdf(+)] )
 	collada? ( >=media-libs/opencollada-1.6.68 )
-	color-management? ( media-libs/opencolorio )
+	color-management? ( <media-libs/opencolorio-2.0.0 )
 	cuda? ( dev-util/nvidia-cuda-toolkit:= )
-	ffmpeg? ( media-video/ffmpeg:=[x264,mp3,encode,theora,jpeg2k?] )
+	ffmpeg? ( media-video/ffmpeg:=[x264,mp3,encode,theora,jpeg2k,vpx,vorbis,opus,xvid] )
```

Comment:
I've added those as hard requirements because those are the codecs Blender expects to be installed.

There are still quite a few users reporting bugs in the blender tracker while using these codecs.
(I had a very fun time hunting down and fixing some reported mpegts video bugs recently... -_- )
However I do agree that personally, I don't either use these older codecs much at all.

Basically I added these and hard requirements as I don't think we can add "strongly suggested" use flags to dependencies, right?

If you want to continue this conversation, we can do some over in the new pull request #21213